python/tests/integration_tests/test_llm_evaluator.py

```diff
@@ -15,7 +15,7 @@ def test_llm_evaluator_init() -> None:
             key="vagueness",
             choices=["Y", "N"],
             description="Whether the response is vague. Y for yes, N for no.",
-            include_explanation=True,
```

Comment:
Should keep a couple of these tests with include_explanation around for backward compat testing

Reply:
added some backward compatibility tests